dotnet/src/Connectors/Connectors.Memory.AzureCognitiveSearch/AzureCognitiveSearchMemory.cs

```diff
@@ -1,355 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-
-using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Runtime.CompilerServices;
-using System.Text;
-using System.Text.RegularExpressions;
-using System.Threading;
-using System.Threading.Tasks;
-using Azure;
-using Azure.Core;
-using Azure.Search.Documents;
-using Azure.Search.Documents.Indexes;
-using Azure.Search.Documents.Indexes.Models;
-using Azure.Search.Documents.Models;
-using Microsoft.SemanticKernel.Memory;
-
-namespace Microsoft.SemanticKernel.Connectors.Memory.AzureCognitiveSearch;
-
-/// <summary>
-/// Semantic Memory implementation using Azure Cognitive Search.
-/// For more information about Azure Cognitive Search see https://learn.microsoft.com/azure/search/search-what-is-azure-search
-/// </summary>
-public class AzureCognitiveSearchMemory : ISemanticTextMemory
```

Comment:
Just heads up: We stopped removing and renaming classes and their public members, if possible and if they are no longer needed and started marking them as obsolete instead to notify devs. using SK about upcoming changes and giving them time to adapt their code to use a new API.